civic-me/civic-me-profile.md

```diff
@@ -0,0 +1,33 @@
+## civic.me profile library
+
+With [civic.me](https://civic.me) users can publish a public profile, with a custom name, headline and profile image selected from the user's NFTs. The [Civic.me profile](https://www.npmjs.com/package/@civic/profile) library is provided to allow for easy retrieval of this profile data. By supplying a wallet address, [did](https://did.civic.com/) or (SNS name)[https://naming.bonfida.org/], a user's public civic.me profile data can easily be retrieved.
```

Comment:
nit: It's a .sol name or domain, SNS is the naming service, of which there may be several instances. Civic.me only supports the .sol one run by bonfida.